apps/calsys/values-tucson-teststand.yaml

```diff
@@ -3,5 +3,15 @@ namespace: calsys
 cscs:
   - genericcamera1
   - gcheaderservice1
+  - electrometer101
```

Comment:
All of these entries need to also be put under the `isSim` parameter since they are normally hardware driven. See the `eas` app for an example.